scipy-stubs/fft/_realtransforms.pyi

```diff
@@ -1,82 +1,84 @@
-from scipy._typing import Untyped
+from numpy import float64, generic
+from numpy.typing import NDArray
```

Comment:
conventionally `numpy` is imported as module with alias `np`, and `numpy.typing` as `npt`

Reply:
this is also why (and only why) the CI is failing btw